lib/binary_to_decimal.rb

```diff
@@ -1,9 +1,10 @@
-# A method named `binary_to_decimal` that receives as input an array of size 8.
-# The array is randomly filled with 0’s and 1’s.
-# The most significant bit is at index 0.
-# The least significant bit is at index 7.
-# Calculate  and return the decimal value for this binary number using
-# the algorithm you devised in class.
 def binary_to_decimal(binary_array)
-  raise NotImplementedError
-end
+  i = 7
```

Comment:
Minor:  I suggest making 7 a constant or using the array's length -1, just to avoid hardcoding numbers in.